hardware/raspberrypi/power/README.md

```diff
@@ -26,6 +26,14 @@ The power requirements of the Raspberry Pi increase as you make use of the vario
 
 If you need to connect a USB device that will take the power requirements above the values specified in the table above, then you must connect it to an externally-powered USB hub.
 
+## Power Supply Warnings
+
+On all models of Raspberry Pi since the Raspberry Pi B+ (2014) there is voltage detection circuitry that will detect if the supply voltage drops below 4.63v (+/- 5%). This will result in a [warning icon](../../../configuration/warning-icons.md) being displayed on all attached displays and an entry being being placed in the kernel log. 
```

Comment:
"placed in the kernel log" -> "added to the kernel log" ?
Also, is it worth changing "voltage detection circuitry" to "low-voltage detection circuitry" ?